### ISSUE TITLE
Feedback metadata is too limited

### ISSUE THREAD

**Issue by mnjongerius** [mopinion/mopinion-sdk-react-native#16]
When I get feedback submitted from the app, the metadata is get with it is very limited:

![image](https://user-images.githubusercontent.com/4756828/84470359-7c0ad780-ac83-11ea-9f32-e043bddc7dde.png)

Expected metadata:
![image](https://user-images.githubusercontent.com/4756828/84470413-9ba20000-ac83-11ea-9ab3-affb8e9f844a.png)

**Comment by stevenkuipers** [comment]
Hi @mnjongerius,

Thanks for contacting us. We are looking into why you are missing this metadata.

Can you tell us which SDK version and what version of RN you are using?
And just to be sure, did you link and install all assets and modules?

**Comment by mnjongerius** [comment]
hi @stevenkuipers,
```
"mopinion-react-native-sdk": "^1.2.1",
"react-native": "0.61.4",
```
SDK is installed as documented

_Edited: typo_

**Comment by stevenkuipers** [comment]
Can you please update to the latest version of the SDK, version 1.2.2?

Also can you contact us at support@mopinion.com, I would like to set up a call with you if that is okay?

**Comment by stevenkuipers** [comment]
We just released a new version: 1.2.3 which addresses this issue. 
The user agent related metadata will be available after updating to the latest version.
